features/step_definitions/create_menu_page.rb

```diff
@@ -30,3 +30,12 @@
     FactoryGirl.create(:menu, menu)
   end
 end
+
+
+And(/^that following menus for "([^"]*)" exist$/) do |name, table|
+  @restaurant = Restaurant.find_by(name: name)
+  table.hashes.each do |menu|
+    #FactoryGirl.create(:menu, menu.merge!(restaurant: @restaurant))
```

Comment:
We will actually use that line after the last merge that sets up the association between Restaurant and Menu